src/app/component/project-option/export/export.component.ts

```diff
@@ -1,5 +1,6 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, Output, EventEmitter } from '@angular/core';
```

Comment:
Output is unused I guess

Reply:
Ahh yes, I missed this. Good spot, thanks.